frontend/src/console/models.ts

```diff
@@ -1,12 +1,16 @@
-import { OpenEdXInstanceDeploymentStatusStatusEnum } from 'ocim-client';
+import {
+  OpenEdXInstanceDeploymentStatusStatusEnum,
+  ThemeSchemaSerializerGenerator
+} from 'ocim-client';
 
 export interface InstanceSettingsModel {
-  [key: string]: string | number;
+  [key: string]: string | number | undefined | ThemeSchemaSerializerGenerator;
```

Comment:
So, this is Typescript, and to define interfaces we need to set explicit types to each variable. For example, if I want to add a variable that will only hold a string I can use:
```
interface InstanceSettingsModel {
  variableName: string;
}
```
In this case, this line is more generic: it means that any variable inside that interface, needs to have a `string` typed key, and only contain one of the values defined after it. For example:
```
interface InstanceSettingsModel {
  [key: string]: string | number;
  // If I try to add a new variable that is not from one of the types above, the compilation fails
  wrongDefinition: ThemeSchemaSerializerGenerator;
  // If I add a variable that is allowed, then the compilation succeeds
  correctDefinition: string;
```
In any case, this line of code is only required so that I can define that all keys should be strings, so I can replace it with the code below to have the same effect:
```
  [key: string]: any | undefined;
```

Reply:
Thanks for the explanation! I prefer definition with "any" at the end of your comment. Should we allow an undefined value? Isn't there always a theme enabled in an OpenEDX instance?

Reply:
Not always, we need to leave the `undefined` there because some clients won't have any theming setup by the time this lands, and if the backend returns an undefined value while we're expecting a defined one the frontend will crash.

Reply:
Changed to `[key: string]: any | undefined;`